### PR TITLE
Adds a Dockerfile for Aqueduct

### PR DESCRIPTION
**Pull request by hsubbaraj-spiral**
## Describe your changes and why you are making these changes
Packages Aqueduct into a Dockerfile. Separating this from the Helm chart PR. 
## Related issue number (if any)
https://linear.app/aqueducthq/issue/ENG-1783/package-aqueduct-into-docker-container
## Loom demo (if any)
```
(base) ubuntu@ip-172-31-46-200:~/aqueduct/src/dockerfiles$ docker run aqueducthq/aqueduct:0.0.16
Updating UI version to 0.0.16...
UI: 100%|████████████████████████████████████████████████| 3556.0/3556.0 [00:00<00:00, 94242.71KB/s]
Updated configurations.
Updating server version to 0.0.16...
Operating system is Linux with architecture amd64.
Downloading server binaries...
Server: 100%|██████████████████████████████████████████| 50508.0/50508.0 [00:00<00:00, 99057.15KB/s]
Executor: 100%|████████████████████████████████████████| 48460.0/48460.0 [00:00<00:00, 82032.38KB/s]
Migrator: 100%|███████████████████████████████████████| 23720.0/23720.0 [00:00<00:00, 100677.90KB/s]
Downloading integration set up scripts...
100%|███████████████████████████████████████████████████████████| 4.0/4.0 [00:00<00:00, 6681.49KB/s]
100%|███████████████████████████████████████████████████████████| 4.0/4.0 [00:00<00:00, 7509.94KB/s]
Downloaded server binaries...
time="2022-10-04T19:58:37Z" level=info msg="Executing query: SELECT version,dirty,name FROM schema_version ORDER BY version DESC LIMIT 1;"
time="2022-10-04T19:58:37Z" level=info msg="Starting migrate up to schema version 1."
time="2022-10-04T19:58:37Z" level=info msg="Executing query: \nCREATE TABLE IF NOT EXISTS schema_version (\n    version INTEGER NOT NULL PRIMARY KEY,\n    dirty BOOL NOT NULL,\n    name TEXT NOT NULL\n);\n\n-- The schema_version record for v000001 needs to be explicitly inserted.\n-- Only insert the record if it doesn't already exist.\nINSERT OR IGNORE INTO schema_version VALUES (1, 1, 'base');\n\nCREATE TABLE IF NOT EXISTS app_user (\n    id BLOB NOT NULL PRIMARY KEY,\n    email TEXT NOT NULL UNIQUE,\n    organization_id TEXT NOT NULL,\n    role TEXT NOT NULL,\n    api_key TEXT NOT NULL UNIQUE,\n    auth0_id TEXT NOT NULL UNIQUE\n);\n\nCREATE TABLE IF NOT EXISTS integration (\n    id BLOB NOT NULL PRIMARY KEY,\n    organization_id TEXT NOT NULL,\n    service TEXT NOT NULL,\n    name TEXT NOT NULL,\n    config BLOB NOT NULL,\n    created_at DATETIME NOT NULL,\n    validated BOOL NOT NULL\n);\n\nCREATE TABLE IF NOT EXISTS notification (\n    id BLOB NOT NULL PRIMARY KEY,\n    receiver_id BLOB NOT NULL REFERENCES app_user (id),\n    content TEXT NOT NULL,\n    status TEXT NOT NULL,\n    level TEXT NOT NULL,\n    association BLOB NOT NULL,\n    created_at DATETIME NOT NULL\n);\n\nCREATE TABLE IF NOT EXISTS workflow (\n    id BLOB NOT NULL PRIMARY KEY,\n    user_id BLOB NOT NULL REFERENCES app_user (id),\n    name TEXT NOT NULL,\n    description TEXT NOT NULL,\n    schedule BLOB NOT NULL,\n    created_at DATETIME NOT NULL,\n    UNIQUE (user_id, name)\n);\n\nCREATE TABLE IF NOT EXISTS workflow_dag (\n    id BLOB NOT NULL PRIMARY KEY,\n    workflow_id BLOB NOT NULL REFERENCES workflow (id),\n    s3_config BLOB NOT NULL,\n    created_at DATETIME NOT NULL\n);\n\nCREATE TABLE IF NOT EXISTS operator (\n    id BLOB NOT NULL PRIMARY KEY,\n    name TEXT NOT NULL,\n    description TEXT NOT NULL,\n    spec BLOB NOT NULL\n);\n\nCREATE TABLE IF NOT EXISTS artifact (\n    id BLOB NOT NULL PRIMARY KEY,\n    name TEXT NOT NULL,\n    description TEXT NOT NULL,\n    spec BLOB NOT NULL\n);\n\nCREATE TABLE IF NOT EXISTS workflow_dag_edge (\n    workflow_dag_id BLOB NOT NULL REFERENCES workflow_dag (id),\n    type TEXT NOT NULL,\n    from_id BLOB NOT NULL,\n    to_id BLOB NOT NULL,\n    idx INTEGER NOT NULL\n);\n\nCREATE TABLE IF NOT EXISTS workflow_dag_result (\n    id BLOB NOT NULL PRIMARY KEY,\n    workflow_dag_id BLOB NOT NULL REFERENCES workflow_dag (id),\n    status TEXT NOT NULL,\n    created_at DATETIME NOT NULL\n);\n\nCREATE TABLE IF NOT EXISTS operator_result (\n    id BLOB NOT NULL PRIMARY KEY,\n    workflow_dag_result_id BLOB NOT NULL REFERENCES workflow_dag_result (id),\n    operator_id BLOB NOT NULL REFERENCES operator (id),\n    status TEXT NOT NULL,\n    metadata BLOB\n);\n\nCREATE TABLE IF NOT EXISTS artifact_result (\n    id BLOB NOT NULL PRIMARY KEY,\n    workflow_dag_result_id BLOB NOT NULL REFERENCES workflow_dag_result (id),\n    artifact_id BLOB NOT NULL REFERENCES artifact (id),\n    content_path TEXT NOT NULL,\n    status TEXT NOT NULL,\n    metadata BLOB\n);\n\nCREATE TABLE IF NOT EXISTS workflow_watcher (\n    workflow_id BLOB NOT NULL REFERENCES workflow (id),\n    user_id BLOB NOT NULL REFERENCES app_user (id),\n    PRIMARY KEY (workflow_id, user_id)\n);\n"
time="2022-10-04T19:58:37Z" level=info msg="Executing query: UPDATE schema_version SET dirty = $1 WHERE version = $2 RETURNING version,dirty,name; with args: [false 1]"
time="2022-10-04T19:58:37Z" level=info msg="Completed migrate up to schema version 1."
time="2022-10-04T19:58:37Z" level=info msg="Starting migrate up to schema version 2."
time="2022-10-04T19:58:37Z" level=info msg="Executing query: SELECT version,dirty,name FROM schema_version WHERE version = 2;"
time="2022-10-04T19:58:37Z" level=info msg="Executing query: INSERT INTO schema_version (version, dirty, name) VALUES ($1, $2, $3) RETURNING version,dirty,name; with args: [2 true add integration.user_id]"
time="2022-10-04T19:58:37Z" level=info msg="Executing query: \nALTER TABLE integration ADD COLUMN user_id BLOB REFERENCES app_user (id);\n"
time="2022-10-04T19:58:37Z" level=info msg="Executing query: UPDATE schema_version SET dirty = $1 WHERE version = $2 RETURNING version,dirty,name; with args: [false 2]"
time="2022-10-04T19:58:37Z" level=info msg="Completed migrate up to schema version 2."
time="2022-10-04T19:58:37Z" level=info msg="Starting migrate up to schema version 3."
time="2022-10-04T19:58:37Z" level=info msg="Executing query: SELECT version,dirty,name FROM schema_version WHERE version = 3;"
time="2022-10-04T19:58:37Z" level=info msg="Executing query: INSERT INTO schema_version (version, dirty, name) VALUES ($1, $2, $3) RETURNING version,dirty,name; with args: [3 true add workflow_dag.storage_config]"
time="2022-10-04T19:58:37Z" level=info msg="Executing query: \nALTER TABLE workflow_dag\nADD COLUMN storage_config BLOB;\n"
time="2022-10-04T19:58:37Z" level=info msg="Executing query: UPDATE schema_version SET dirty = $1 WHERE version = $2 RETURNING version,dirty,name; with args: [false 3]"
time="2022-10-04T19:58:37Z" level=info msg="Completed migrate up to schema version 3."
time="2022-10-04T19:58:37Z" level=info msg="Starting migrate up to schema version 4."
time="2022-10-04T19:58:37Z" level=info msg="Executing query: SELECT version,dirty,name FROM schema_version WHERE version = 4;"
time="2022-10-04T19:58:37Z" level=info msg="Executing query: INSERT INTO schema_version (version, dirty, name) VALUES ($1, $2, $3) RETURNING version,dirty,name; with args: [4 true backfill workflow_dag.storage_config and operator.spec.storage_path]"
time="2022-10-04T19:58:37Z" level=info msg="Executing query: SELECT id, s3_config FROM workflow_dag;"
time="2022-10-04T19:58:37Z" level=info msg="Executing query: SELECT id, spec FROM operator;"
time="2022-10-04T19:58:37Z" level=info msg="Executing query: UPDATE schema_version SET dirty = $1 WHERE version = $2 RETURNING version,dirty,name; with args: [false 4]"
time="2022-10-04T19:58:37Z" level=info msg="Completed migrate up to schema version 4."
time="2022-10-04T19:58:37Z" level=info msg="Starting migrate up to schema version 5."
time="2022-10-04T19:58:37Z" level=info msg="Executing query: SELECT version,dirty,name FROM schema_version WHERE version = 5;"
time="2022-10-04T19:58:37Z" level=info msg="Executing query: INSERT INTO schema_version (version, dirty, name) VALUES ($1, $2, $3) RETURNING version,dirty,name; with args: [5 true add not null constraint to workflow_dag.storage_config]"
time="2022-10-04T19:58:37Z" level=info msg="Executing query: \nBEGIN TRANSACTION;\n\nALTER TABLE workflow_dag RENAME TO tmp_workflow_dag;\n\nCREATE TABLE workflow_dag (\n    id BLOB NOT NULL PRIMARY KEY,\n    workflow_id BLOB NOT NULL REFERENCES workflow (id),\n    s3_config BLOB NOT NULL,\n    created_at DATETIME NOT NULL,\n    storage_config BLOB NOT NULL\n);\n\nINSERT INTO workflow_dag(id, workflow_id, s3_config, created_at, storage_config)\nSELECT id, workflow_id, s3_config, created_at, storage_config\nFROM tmp_workflow_dag;\n\nDROP TABLE tmp_workflow_dag;\n\nCOMMIT;\n"
time="2022-10-04T19:58:37Z" level=info msg="Executing query: UPDATE schema_version SET dirty = $1 WHERE version = $2 RETURNING version,dirty,name; with args: [false 5]"
time="2022-10-04T19:58:37Z" level=info msg="Completed migrate up to schema version 5."
time="2022-10-04T19:58:37Z" level=info msg="Starting migrate up to schema version 6."
time="2022-10-04T19:58:37Z" level=info msg="Executing query: SELECT version,dirty,name FROM schema_version WHERE version = 6;"
time="2022-10-04T19:58:37Z" level=info msg="Executing query: INSERT INTO schema_version (version, dirty, name) VALUES ($1, $2, $3) RETURNING version,dirty,name; with args: [6 true add workflow.retention_policy]"
time="2022-10-04T19:58:37Z" level=info msg="Executing query: \nALTER TABLE workflow\nADD COLUMN retention_policy BLOB NOT NULL;\n"
time="2022-10-04T19:58:37Z" level=info msg="Executing query: UPDATE schema_version SET dirty = $1 WHERE version = $2 RETURNING version,dirty,name; with args: [false 6]"
time="2022-10-04T19:58:37Z" level=info msg="Completed migrate up to schema version 6."
time="2022-10-04T19:58:37Z" level=info msg="Starting migrate up to schema version 7."
time="2022-10-04T19:58:37Z" level=info msg="Executing query: SELECT version,dirty,name FROM schema_version WHERE version = 7;"
time="2022-10-04T19:58:37Z" level=info msg="Executing query: INSERT INTO schema_version (version, dirty, name) VALUES ($1, $2, $3) RETURNING version,dirty,name; with args: [7 true add primary key constraint to workflow_dag_edge on workflow_dag_id, from_id, to_id]"
time="2022-10-04T19:58:37Z" level=info msg="Executing query: \nBEGIN TRANSACTION;\n\nALTER TABLE workflow_dag_edge RENAME TO tmp_workflow_dag_edge;\n\nCREATE TABLE workflow_dag_edge (\n    workflow_dag_id BLOB NOT NULL REFERENCES workflow_dag (id),\n    type TEXT NOT NULL,\n    from_id BLOB NOT NULL,\n    to_id BLOB NOT NULL,\n    idx INTEGER NOT NULL,\n\tPRIMARY KEY (workflow_dag_id, from_id, to_id)\n);\n\nINSERT INTO workflow_dag_edge(workflow_dag_id, type, from_id, to_id, idx)\nSELECT workflow_dag_id, type, from_id, to_id, idx\nFROM tmp_workflow_dag_edge;\n\nDROP TABLE tmp_workflow_dag_edge;\n\nCOMMIT;\n"
time="2022-10-04T19:58:37Z" level=info msg="Executing query: UPDATE schema_version SET dirty = $1 WHERE version = $2 RETURNING version,dirty,name; with args: [false 7]"
time="2022-10-04T19:58:37Z" level=info msg="Completed migrate up to schema version 7."
time="2022-10-04T19:58:37Z" level=info msg="Starting migrate up to schema version 8."
time="2022-10-04T19:58:37Z" level=info msg="Executing query: SELECT version,dirty,name FROM schema_version WHERE version = 8;"
time="2022-10-04T19:58:37Z" level=info msg="Executing query: INSERT INTO schema_version (version, dirty, name) VALUES ($1, $2, $3) RETURNING version,dirty,name; with args: [8 true delete outdated s3_config column]"
time="2022-10-04T19:58:37Z" level=info msg="Executing query: \nALTER TABLE workflow_dag\nDROP COLUMN s3_config;\n"
time="2022-10-04T19:58:37Z" level=info msg="Executing query: UPDATE schema_version SET dirty = $1 WHERE version = $2 RETURNING version,dirty,name; with args: [false 8]"
time="2022-10-04T19:58:37Z" level=info msg="Completed migrate up to schema version 8."
time="2022-10-04T19:58:37Z" level=info msg="Starting migrate up to schema version 9."
time="2022-10-04T19:58:37Z" level=info msg="Executing query: SELECT version,dirty,name FROM schema_version WHERE version = 9;"
time="2022-10-04T19:58:37Z" level=info msg="Executing query: INSERT INTO schema_version (version, dirty, name) VALUES ($1, $2, $3) RETURNING version,dirty,name; with args: [9 true backfill metadata in artifact_results]"
time="2022-10-04T19:58:37Z" level=info msg="Executing query: SELECT id, metadata FROM artifact_result;"
time="2022-10-04T19:58:37Z" level=info msg="Executing query: UPDATE schema_version SET dirty = $1 WHERE version = $2 RETURNING version,dirty,name; with args: [false 9]"
time="2022-10-04T19:58:37Z" level=info msg="Completed migrate up to schema version 9."
time="2022-10-04T19:58:37Z" level=info msg="Starting migrate up to schema version 10."
time="2022-10-04T19:58:37Z" level=info msg="Executing query: SELECT version,dirty,name FROM schema_version WHERE version = 10;"
time="2022-10-04T19:58:37Z" level=info msg="Executing query: INSERT INTO schema_version (version, dirty, name) VALUES ($1, $2, $3) RETURNING version,dirty,name; with args: [10 true add exec state column to operator_result]"
time="2022-10-04T19:58:37Z" level=info msg="Executing query: \nALTER TABLE operator_result\nADD COLUMN execution_state BLOB;\n"
time="2022-10-04T19:58:37Z" level=info msg="Executing query: UPDATE schema_version SET dirty = $1 WHERE version = $2 RETURNING version,dirty,name; with args: [false 10]"
time="2022-10-04T19:58:37Z" level=info msg="Completed migrate up to schema version 10."
time="2022-10-04T19:58:37Z" level=info msg="Starting migrate up to schema version 11."
time="2022-10-04T19:58:37Z" level=info msg="Executing query: SELECT version,dirty,name FROM schema_version WHERE version = 11;"
time="2022-10-04T19:58:37Z" level=info msg="Executing query: INSERT INTO schema_version (version, dirty, name) VALUES ($1, $2, $3) RETURNING version,dirty,name; with args: [11 true backfill exec state column in operator_result]"
time="2022-10-04T19:58:37Z" level=info msg="Executing query: SELECT id, status, metadata FROM operator_result;"
time="2022-10-04T19:58:37Z" level=info msg="Executing query: UPDATE schema_version SET dirty = $1 WHERE version = $2 RETURNING version,dirty,name; with args: [false 11]"
time="2022-10-04T19:58:37Z" level=info msg="Completed migrate up to schema version 11."
time="2022-10-04T19:58:37Z" level=info msg="Starting migrate up to schema version 12."
time="2022-10-04T19:58:37Z" level=info msg="Executing query: SELECT version,dirty,name FROM schema_version WHERE version = 12;"
time="2022-10-04T19:58:37Z" level=info msg="Executing query: INSERT INTO schema_version (version, dirty, name) VALUES ($1, $2, $3) RETURNING version,dirty,name; with args: [12 true remove metadata in operator_result]"
time="2022-10-04T19:58:37Z" level=info msg="Executing query: \nALTER TABLE operator_result\nDROP COLUMN metadata;\n"
time="2022-10-04T19:58:37Z" level=info msg="Executing query: UPDATE schema_version SET dirty = $1 WHERE version = $2 RETURNING version,dirty,name; with args: [false 12]"
time="2022-10-04T19:58:37Z" level=info msg="Completed migrate up to schema version 12."
time="2022-10-04T19:58:37Z" level=info msg="Starting migrate up to schema version 13."
time="2022-10-04T19:58:37Z" level=info msg="Executing query: SELECT version,dirty,name FROM schema_version WHERE version = 13;"
time="2022-10-04T19:58:37Z" level=info msg="Executing query: INSERT INTO schema_version (version, dirty, name) VALUES ($1, $2, $3) RETURNING version,dirty,name; with args: [13 true add workflow_dag.engine_config]"
time="2022-10-04T19:58:37Z" level=info msg="Executing query: Transaction BEGIN"
time="2022-10-04T19:58:37Z" level=info msg="Executing query: ALTER TABLE workflow_dag ADD COLUMN engine_config BLOB;"
time="2022-10-04T19:58:37Z" level=info msg="Executing query: UPDATE workflow_dag SET engine_config = $1 WHERE engine_config IS NULL; with args: [0xc00000f248]"
time="2022-10-04T19:58:37Z" level=info msg="Executing query: \n\tALTER TABLE workflow_dag RENAME TO tmp_workflow_dag;\n\n\tCREATE TABLE workflow_dag (\n\t\tid BLOB NOT NULL PRIMARY KEY,\n\t\tworkflow_id BLOB NOT NULL REFERENCES workflow (id),\n\t\tcreated_at DATETIME NOT NULL,\n\t\tstorage_config BLOB NOT NULL,\n\t\tengine_config BLOB NOT NULL\n\t);\n\n\tINSERT INTO workflow_dag(id, workflow_id, created_at, storage_config, engine_config)\n\tSELECT id, workflow_id, created_at, storage_config, engine_config\n\tFROM tmp_workflow_dag;\n\n\tDROP TABLE tmp_workflow_dag;\n\t"
time="2022-10-04T19:58:37Z" level=info msg="Executing query: Transaction COMMIT"
time="2022-10-04T19:58:37Z" level=info msg="Executing query: UPDATE schema_version SET dirty = $1 WHERE version = $2 RETURNING version,dirty,name; with args: [false 13]"
time="2022-10-04T19:58:37Z" level=info msg="Completed migrate up to schema version 13."
time="2022-10-04T19:58:37Z" level=info msg="Starting migrate up to schema version 14."
time="2022-10-04T19:58:37Z" level=info msg="Executing query: SELECT version,dirty,name FROM schema_version WHERE version = 14;"
time="2022-10-04T19:58:37Z" level=info msg="Executing query: INSERT INTO schema_version (version, dirty, name) VALUES ($1, $2, $3) RETURNING version,dirty,name; with args: [14 true add exec state column to artifact result]"
time="2022-10-04T19:58:37Z" level=info msg="Executing query: \nALTER TABLE artifact_result \nADD COLUMN execution_state BLOB;\n"
time="2022-10-04T19:58:37Z" level=info msg="Executing query: UPDATE schema_version SET dirty = $1 WHERE version = $2 RETURNING version,dirty,name; with args: [false 14]"
time="2022-10-04T19:58:37Z" level=info msg="Completed migrate up to schema version 14."
preferences: {}
kind: Config

clusters:
- cluster:
    server: https://C42FD95719DD8374B16C8F6DA4A1F3A1.gr7.us-east-2.eks.amazonaws.com
    certificate-authority-data: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUM1ekNDQWMrZ0F3SUJBZ0lCQURBTkJna3Foa2lHOXcwQkFRc0ZBREFWTVJNd0VRWURWUVFERXdwcmRXSmwKY201bGRHVnpNQjRYRFRJeU1UQXdOREF3TkRneE4xb1hEVE15TVRBd01UQXdORGd4TjFvd0ZURVRtime="2022-10-04T19:58:37Z" level=info msg="Executing query: SELECT version,dirty,name FROM schema_version WHERE version = 15;"
time="2022-10-04T19:58:37Z" level=info msg="Executing query: INSERT INTO schema_version (version, dirty, name) VALUES ($1, $2, $3) RETURNING version,dirty,name; with args: [15 true backfill exec state column in artifact result]"
time="2022-10-04T19:58:37Z" level=info msg="Executing query: \n\t\tSELECT artifact_result.id, operator_result.execution_state\n\t\tFROM operator_result\n\t\tINNER JOIN operator ON operator_result.operator_id=operator.id\n\t\tINNER JOIN workflow_dag_edge ON operator.id=workflow_dag_edge.from_id\n\t\tINNER JOIN artifact ON workflow_dag_edge.to_id=artifact.id\n\t\tINNER JOIN artifact_result ON artifact.id=artifact_result.artifact_id;\n\t"
time="2022-10-04T19:58:37Z" level=info msg="Executing query: UPDATE schema_version SET dirty = $1 WHERE version = $2 RETURNING version,dirty,name; with args: [false 15]"
time="2022-10-04T19:58:37Z" level=info msg="Completed migrate up to schema version 15."
time="2022-10-04T19:58:37Z" level=info msg="Starting migrate up to schema version 16."
time="2022-10-04T19:58:37Z" level=info msg="Executing query: SELECT version,dirty,name FROM schema_version WHERE version = 16;"
time="2022-10-04T19:58:37Z" level=info msg="Executing query: INSERT INTO schema_version (version, dirty, name) VALUES ($1, $2, $3) RETURNING version,dirty,name; with args: [16 true add type column to artifact]"
time="2022-10-04T19:58:37Z" level=info msg="Executing query: \nALTER TABLE artifact\nADD COLUMN type TEXT NOT NULL DEFAULT 'untyped';\n"
time="2022-10-04T19:58:37Z" level=info msg="Executing query: SELECT id, spec FROM artifact;"
time="2022-10-04T19:58:37Z" level=info msg="A total of 0 artifact results have been migrated."
time="2022-10-04T19:58:37Z" level=info msg="Executing query: \nALTER TABLE artifact \nDROP COLUMN spec;\n"
time="2022-10-04T19:58:37Z" level=info msg="Executing query: UPDATE schema_version SET dirty = $1 WHERE version = $2 RETURNING version,dirty,name; with args: [false 16]"
time="2022-10-04T19:58:37Z" level=info msg="Completed migrate up to schema version 16."
time="2022-10-04T19:58:37Z" level=info msg="Starting migrate up to schema version 17."
time="2022-10-04T19:58:37Z" level=info msg="Executing query: SELECT version,dirty,name FROM schema_version WHERE version = 17;"
time="2022-10-04T19:58:37Z" level=info msg="Executing query: INSERT INTO schema_version (version, dirty, name) VALUES ($1, $2, $3) RETURNING version,dirty,name; with args: [17 true add canceled status to results]"
time="2022-10-04T19:58:37Z" level=info msg="Executing query: SELECT id, execution_state FROM artifact_result WHERE status=\"pending\";"
time="2022-10-04T19:58:37Z" level=info msg="Executing query: SELECT id, execution_state FROM operator_result WHERE status=\"pending\";"
time="2022-10-04T19:58:37Z" level=info msg="Executing query: SELECT id, execution_state FROM artifact_result WHERE status=\"failed\";"
time="2022-10-04T19:58:37Z" level=info msg="Executing query: UPDATE schema_version SET dirty = $1 WHERE version = $2 RETURNING version,dirty,name; with args: [false 17]"
time="2022-10-04T19:58:37Z" level=info msg="Completed migrate up to schema version 17."
time="2022-10-04T19:58:37Z" level=info msg="Starting migrate up to schema version 18."
time="2022-10-04T19:58:37Z" level=info msg="Executing query: SELECT version,dirty,name FROM schema_version WHERE version = 18;"
time="2022-10-04T19:58:37Z" level=info msg="Executing query: INSERT INTO schema_version (version, dirty, name) VALUES ($1, $2, $3) RETURNING version,dirty,name; with args: [18 true add exec state to workflow_dag_result]"
time="2022-10-04T19:58:37Z" level=info msg="Executing query: \nALTER TABLE workflow_dag_result \nADD COLUMN execution_state BLOB;\n"
time="2022-10-04T19:58:37Z" level=info msg="Executing query: SELECT id, status, created_at FROM workflow_dag_result;"
time="2022-10-04T19:58:37Z" level=info msg="Executing query: UPDATE schema_version SET dirty = $1 WHERE version = $2 RETURNING version,dirty,name; with args: [false 18]"
time="2022-10-04T19:58:37Z" level=info msg="Completed migrate up to schema version 18."
time="2022-10-04T19:58:37Z" level=info msg="Starting migrate up to schema version 19."
time="2022-10-04T19:58:37Z" level=info msg="Executing query: SELECT version,dirty,name FROM schema_version WHERE version = 19;"
time="2022-10-04T19:58:37Z" level=info msg="Executing query: INSERT INTO schema_version (version, dirty, name) VALUES ($1, $2, $3) RETURNING version,dirty,name; with args: [19 true add serialization type and value to param op]"
time="2022-10-04T19:58:37Z" level=info msg="Executing query: SELECT id, spec FROM operator;"
time="2022-10-04T19:58:37Z" level=info msg="Executing query: UPDATE schema_version SET dirty = $1 WHERE version = $2 RETURNING version,dirty,name; with args: [false 19]"
time="2022-10-04T19:58:37Z" level=info msg="Completed migrate up to schema version 19."
time="2022-10-04T19:58:37Z" level=info msg="Checking current schema version..."
time="2022-10-04T19:58:37Z" level=info msg="Executing query: SELECT version,dirty,name FROM schema_version ORDER BY version DESC LIMIT 1;"
time="2022-10-04T19:58:37Z" level=info msg="The current schema version is 000019."
Finished initializing Aqueduct base directory.

***************************************************
Your API Key: O0CHXD3TRJA5IGQ6SEW2NYMBP1KLZF48

The Web UI and the backend server are accessible at: http://3.130.165.53:8080/login?apiKey=O0CHXD3TRJA5IGQ6SEW2NYMBP1KLZF48
***************************************************
```

## Checklist before requesting a review
- [x] I have created a descriptive PR title. The PR title should complete the sentence "This PR...".
- [x] I have performed a self-review of my code.
- [ ] I have included a small demo of the changes. For the UI, this would be a screenshot or a Loom video.
- [ ] If this is a new feature, I have added unit tests and integration tests.
- [ ] I have run the integration tests locally and they are passing.
- [ ] I have run the linter script locally (See `python3 scripts/run_linters.py -h` for usage).
- [ ] All features on the UI continue to work correctly.
- [ ] Added one of the following CI labels:
    - `run_integration_test`: Runs integration tests
    - `skip_integration_test`: Skips integration tests (Should be used when changes are ONLY documentation/UI)

### CODE REVIEW

File: src/dockerfiles/aqueduct.dockerfile [hunk 1]
```diff
@@ -0,0 +1,12 @@
+FROM python:3.9
+
+MAINTAINER Aqueduct <hello@aqueducthq.com> version: 0.0.1
+
+USER root
+
+ENV PYTHONUNBUFFERED 1
+
+RUN pip install aqueduct-ml==0.0.16
```
Comment (saurav-c) [inline src/dockerfiles/aqueduct.dockerfile:9]: nit: should the version be passed in as an arg?

**Review by saurav-c — APPROVED** [review]
Left a comment, otherwise LGTM

File: src/dockerfiles/aqueduct.dockerfile (hunk 1 above)
Comment (hsubbaraj-spiral) [reply src/dockerfiles/aqueduct.dockerfile:9]: I think we will version this image with the release.

Comment (hsubbaraj-spiral) [reply src/dockerfiles/aqueduct.dockerfile]: Added the version as a variable.

**Pull request merged**